sphinx/util/texescape.py

```diff
@@ -78,47 +81,38 @@
     ('₉', r'\(\sb{\text{9}}\)'),
 ]
 
-tex_escape_map = {}  # type: Dict[int, str]
-tex_escape_map_without_unicode = {}  # type: Dict[int, str]
-tex_replace_map = {}
-tex_hl_escape_map_new = {}  # type: Dict[int, str]
-tex_hl_escape_map_new_without_unicode = {}  # type: Dict[int, str]
+tex_replace_map = {}  # type: Dict[int, str]
 
+_tex_escape_map = {}  # type: Dict[int, str]
+_tex_escape_map_without_unicode = {}  # type: Dict[int, str]
+_tex_hlescape_map = {}  # type: Dict[int, str]
+_tex_hlescape_map_without_unicode = {}  # type: Dict[int, str]
 
-def get_escape_func(latex_engine: str) -> Callable[[str], str]:
```

Comment:
I determined to drop `get_escape_func()` and add a feature to switch escaping map to `escape()` function itself. I think enough simple and allows to keep compatibility for modules already using `escape()` function.

Reply:
+1 I will review a bit later